packages/roosterjs-content-model/lib/publicTypes/format/ContentModelListItemLevelFormat.ts

```diff
@@ -9,4 +10,5 @@ import { ListTypeFormat } from './formatParts/ListTypeFormat';
 export type ContentModelListItemLevelFormat = ListTypeFormat &
     ListThreadFormat &
     ListMetadataFormat &
+    DisplayFormat &
```

Comment:
I think you don't need to explicitly add DisplayFormat here.

in directionFormatHandler you can just read/write display from the element and translate it to textAlign in Content Model